downstream/modules/platform/proc-install-aap-operator-chatbot.adoc

```diff
@@ -0,0 +1,26 @@
+
+[id="proc-install-operator-chatbot"]
+
+= Installing and configuring the {PlatformNameShort} operator
```

Comment:
Should 'operator" be uppercase? I think it's supposed to be for Kubernetes and OpenShift, not sure about here.

Reply:
OK, we talked and no, "operator" is correct. :)